core/services/keeper/registry1_2_synchronizer_test.go

```diff
@@ -72,8 +75,16 @@ func mockRegistry1_2(
 		Config:  config,
 		Keepers: keeperList,
 	}
-	registryMock.MockResponse("getState", getState).Once()
-	registryMock.MockResponse("getActiveUpkeepIDs", activeUpkeepIDs).Once()
+	ethMock.On("HeaderByNumber", context.Background(), (*big.Int)(nil)).Return(
```

Comment:
nit. you can use mock.Anything here to be a bit general if you want

Reply:
done